src/popup/router/pages/FungibleTokens/TokensPreview.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div class="tokens-preview">
+    <div class="menu">
+      <a class="menu-item" :class="{ active: activeItem === 'tokens' }">
+        <TokenPiles />
+        {{ $t('pages.fungible-tokens.tokens') }}
+      </a>
+    </div>
+    <div class="search" :class="{ focused }">
```

Comment:
```suggestion
    <div class="search">
```
can't be used `:focus-within` selector instead, as in https://github.com/aeternity/superhero-ui/blob/444c38bc7af32c32c31b04095498fc739739aa0a/src/components/MessageInput.vue#L83-L85?